mappings/net/minecraft/util/registry/RegistryKey.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_5321 net/minecraft/util/registry/RegistryKey
+	FIELD field_25136 INSTANCES Ljava/util/Map;
+	FIELD field_25137 registryId Lnet/minecraft/class_2960;
```

Comment:
would name this parentId imo, but up for debate

Reply:
registryId is clearer

Reply:
you can have a registry in both the key and the value idd, it is more about which one is the parent imo

Reply:
What isn't clear to me though with that is that how do we know "parent" refers to a registry?

Reply:
parentRegistryId? sounds long and weird tho

but you don't really have to know if that's a registry, because it is just defining a parent for that entry imo

Reply:
No, just `registryId`

Reply:
Or even just `registry` as this is just a pojo and holds no references.